### PR TITLE
Internalize GetFableModulesFromDir & GetFableModulesFromProject

### PR DESCRIPTION
**Pull request by nojaf**
I had a bit of a hard time following where `evaluateOnly` is being used.
Turns out `CrackerOptions.GetFableModulesFromProject` is not used outside `ProjectCracker`.

### CODE REVIEW

**Review by MangelMaxime — CHANGES_REQUESTED** [review]
If I understand correctly this PR, it is to avoid exposing a `static member`?

File: src/Fable.Compiler/ProjectCracker.fs [hunk 1]
```diff
@@ -84,11 +84,47 @@ type CacheInfo =
         | None -> true
         | Some other -> this.GetTimestamp() > other.GetTimestamp()
 
+/// Combine the `baseDir` with `fable_modules`
+let getFableModulesFromDir (baseDir: string) : string =
+    IO.Path.Combine(baseDir, Naming.fableModules) |> Path.normalizePath
+
+let getFableModulesFromProject
```
[the remark this answers is not in the document]
Comment (nojaf) [reply src/Fable.Compiler/ProjectCracker.fs:91]: ProjectCracker has a signature file, everything that is not listed in a signature will end up being private.

File: src/Fable.Compiler/ProjectCracker.fs [hunk 2]
```diff
@@ -147,38 +183,6 @@ type CrackerOptions(cliArgs: CliArgs, evaluateOnly: bool) =
     static member GetFableModulesFromDir(baseDir: string) : string =
         IO.Path.Combine(baseDir, Naming.fableModules) |> Path.normalizePath
```
[the remark this answers is not in the document]
Comment (nojaf) [reply src/Fable.Compiler/ProjectCracker.fs]: Yes, forgot to remove this one after moving it.

File: src/Fable.Compiler/ProjectCracker.fs (hunk 1 above)
[the remark this answers is not in the document]
Comment (MangelMaxime) [reply src/Fable.Compiler/ProjectCracker.fs:91]: Right I forgot about that

**Pull request merged**